File: src/main/java/src/SistemaGestorTareas/Tarea.java
package SistemaGestorTareas;

public class Tarea {
    private String nombre;
    private boolean completada;

    public Tarea(String nombre) {
        this.nombre = nombre;
        this.completada = false;
    }

    public String getNombre() {
        return nombre;
    }

    public boolean isCompletada() {
        return completada;
    }

    public void marcarComoCompletada() {
        completada = true;
        System.out.println("La tarea '" + nombre + "' ha sido marcada como completada.");
    }

    @Override
    public String toString() {
        return nombre + " - " + (completada ? "Completada" : "Pendiente");
    }
}

File: src/main/kotlin/SistemaGestorTareas/Tarea.kt
package SistemaGestorTareas

class Tarea(val nombre: String) {
    var isCompletada: Boolean = false

    fun marcarComoCompletada() {
        isCompletada = true
        println("La tarea '$nombre' ha sido marcada como completada.")
    }

    override fun toString(): String {
        return nombre + " - " + (if (isCompletada) "Completada" else "Pendiente")
    }
}